SharedFrameworkValidation.cmd

```diff
@@ -1,9 +1,14 @@
 @if not defined _echo @echo off
```

Comment:
mm not really, I just added them there so that it's easier to call them, but it is totally fine to move them under src/SharedFrameworkValidation if you rather have them there.